nwb_conversion_tools/datainterfaces/ecephys/baselfpextractorinterface.py

```diff
@@ -13,6 +13,9 @@
 class BaseLFPExtractorInterface(BaseRecordingExtractorInterface):
     """Primary class for all LFP data interfaces."""
 
+    def __init__(self, **source_data):
+        super().__init__(**source_data)
```

Comment:
```suggestion
    def __init__(self, **source_data):
        super().__init__(**source_data)
```
What does this do that inheriting the `__init__` from the parent doesn't? Note the `AxonaLFPDataInterface` has its own override to the `__init__`, the super should just be needed there instead.